resin/models/device.py

```diff
@@ -530,28 +534,6 @@ def register(self, app_name, uuid):
             endpoint=self.settings.get('pine_endpoint'), login=True
         )
 
-    def restart(self, uuid):
```

Comment:
Why introduce a new entity to the user (i.e. the supervisor)? Also, semantically seems to me as incorrect: you don't want to restart the supervisor, but the application on a specific or all devices.

Is this on parity with the JS SDK (I couldn't find the respective method)?